index.html

```diff
@@ -363,8 +363,9 @@ <h3 class="contact-me">Contact me</h3>
     
             <button class="submit">Get In Touch</button>
```

Comment:
- [ ] I kindly suggest that you add the button interaction for when your button is active.
### EXAMPLE:

**for pressed/active state**
```
button:active {
   background-color: #2230D2;
}
```